src/JPEGView/SettingsProvider.h

```diff
@@ -131,6 +131,11 @@ class CSettingsProvider
 	bool BeepSoundAlert() { return m_bBeepSoundAlert; }
 	double ZoomPauseFactor() { return m_zoomPauseFactor; }  // while internally this is represented in doubles, using a whole number percent simplifies it for the user... configuring doubles is not user friendly at all
 
+	std::vector<double> ParseCustomZoomSteps(LPCTSTR data);
+	bool UseZoomSteps() { return m_bUseZoomSteps;  }

```

Comment:
remove extra but unneeded white space, thanks!  (two spaces before trailing `}`)

Reply:
and then move the two getters up to the other methods which are all just single line getters without adding too much whitespace :)